src/vs/workbench/contrib/codeEditor/browser/saveParticipants.ts

```diff
@@ -29,31 +29,6 @@ import { Disposable } from 'vs/base/common/lifecycle';
 import { IWorkbenchContribution, Extensions as WorkbenchContributionsExtensions, IWorkbenchContributionsRegistry } from 'vs/workbench/common/contributions';
 import { Registry } from 'vs/platform/registry/common/platform';
 import { LifecyclePhase } from 'vs/platform/lifecycle/common/lifecycle';
-import { INotebookService } from 'sql/workbench/services/notebook/browser/notebookService';
```

Comment:
Unsure how this got back in, but this was removed in https://github.com/microsoft/azuredatastudio/pull/8995/files